apps/Info.html

```diff
@@ -164,9 +164,9 @@ <h3 class="text-center h3 mb-2" style="margin-top:8px;margin-bottom:10px;">Infor
 	</div>
 </div>
 
-<div class="text-center text-white bg-dark p-3 mt-2 footer" style="position: static;bottom: 0;width: 100%;">
+<footer class="text-center text-white bg-dark p-3 mt-2">
 	<p class="p">Copyright © 2020 caMicroscope</p>
```

Comment:
Looks good to me!
Just, while you're at it, welcome to 2021!
```suggestion
	<p class="p">Copyright © 2021 caMicroscope</p>
```

Reply:
Haha yes, good suggestion. Adding it😊
Thanks for the review!